guides/source/threading_and_code_execution.md

```diff
@@ -289,7 +289,7 @@ Another example, using Concurrent Ruby:
 ```ruby
 Rails.application.executor.wrap do
   futures = 3.times.collect do |i|
-    Concurrent::Future.execute do
+    Concurrent::Promises.future do
```

Comment:
It looks like we can also replace this usage in the test: https://github.com/rails/rails/blob/0b5c7210af13021aab488adea76cec04a0c218fc/actioncable/test/client_test.rb#L204, would you mind amending the commit adding that change as well? We can keep it all in a single update. Thanks.

Reply:
Good catch, I should've done a global search! updated as well